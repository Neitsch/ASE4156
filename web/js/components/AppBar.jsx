import React from 'react';
import AppBar from 'material-ui/AppBar';
import Toolbar from 'material-ui/Toolbar';
import Typography from 'material-ui/Typography';

export default class App_Bar extends React.Component {
  render() {
    return (
      <AppBar position="static" color="default">
        <Toolbar>
          <Typography>
            Wells Fargo
          </Typography>
        </Toolbar>
      </AppBar>
    );
  }
}
