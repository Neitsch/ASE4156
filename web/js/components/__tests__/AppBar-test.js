/* @lazyspec (remove to manage manually) */
/* eslint-disable */
import AppBar from '../AppBar.jsx';

import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('AppBar', () => {
  it('exists', () => {
    expect(AppBar).toBeTruthy();
  });

  it('renders', () => {
    const comp = <AppBar  />;
    const wrapper = shallow(comp);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
