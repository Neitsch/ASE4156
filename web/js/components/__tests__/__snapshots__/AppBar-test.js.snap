// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar renders 1`] = `
<withStyles(AppBar)
  color="default"
  position="static"
>
  <withStyles(Toolbar)>
    <withStyles(Typography)>
      Wells Fargo
    </withStyles(Typography)>
  </withStyles(Toolbar)>
</withStyles(AppBar)>
`;
